phpunit.xml

```diff
@@ -0,0 +1,5 @@
+<phpunit
```

Comment:
Must be `phpunit.xml.dist`: https://github.com/dunglas/DunglasApiBundle/blob/master/phpunit.xml.dist